docs/README.md

```diff
@@ -6,18 +6,21 @@ If you have Tinkerbell running in your environment, you can use it for CAPT.
 
 Here is the list of required components to use CAPT:
 
-- Existing Tinkerbell installation running at least versions mentioned in [sandbox](https://github.com/tinkerbell/sandbox/commit/be40a7b371fdab947a413e190cfe99b692651d8e).
+- Existing Tinkerbell installation running at least versions mentioned in [sandbox](https://github.com/tinkerbell/sandbox/tree/v0.5.0), this guide assumes deployment using the sandbox with an IP address of 192.168.1.1, so modifications will be needed if Tinkerbell was deployed with a different method or if the IP address is different.
 - A Kubernetes cluster which pods has access to your Tinkerbell instance.
-- A Container registry reachable by both your Kubernetes cluster and your workstation with
-  write access.
-- At least one Hardware available with DHCP IP address configured on first interface and with
-  /dev/vda disk.
+- At least one Hardware available with DHCP IP address configured on first interface and with proper metadata configured
 - `git` binary
 - `tilt` binary
 - `kubectl` binary
 - `clusterctl` binary
 - `go` binary
 
+Required metadata for hardware:
```

Comment:
Can we specify why those fields are mandatory?

Reply:
I created https://github.com/tinkerbell/cluster-api-provider-tink/issues/56 to follow up on this